docs/hcc.md

```diff
@@ -0,0 +1,48 @@
+# git-source-control for Health Connect Cloud
+
+## Development Workflow
+
+git-source-control has two main use modes: Basic and Expert. For Health Connect Cloud, we recommend beginning with basic mode, since it includes all of the necessary functionality for proper source control development while being easy to use for those with little development / source control experience. The recommended workflow for HCC is to create featyre branches, merge them into development, and then git pull the changes in the shared development namespace. The rest of this section explains this workflow more in depth for those with less source control experience. 
+
+### Feature branches
+
+The first step in making changes in Health Connect Cloud using git-source-control is making a feature branch. In order for changes to be tracked by source control properly, each change (also called a feature) should be made on it's own branch, so as to not interfere with other changes, and allow for testing of its effects on the production environment. To create a new feature branch, navigate to the Git UI in the namespace you plan on doing development. 
+
+First, make sure that you are in the development branch. It should be bolded and at the top of the Local Branches in the sidebar. If it is not, as pictured below, click on the "development" branch and press checkout branch.
+
+![Development branch in Git UI Sidebar](images/hcc/developmentsidebar.png)
+
+Once you are in the development branch, press the "+" next the the Local Branches tab, and create a new branch by typing out a name for it. (No spaces or special characters are allowed in branch names).
+
+![Creating a new via sidebar](images/hcc/newbranch.png)
+![Naming new branch](images/hcc/newbranchnaming.png)
+
+Finally, you should click on the new branch you just made in the sidebar, and then press "Push".
+
+![Push your new branch](images/hcc/pushbranch.png)
+
+This makes sure that your branch will be tracked in Gitlab, so that other users can checkout your branch to test your changes on their own namespaces.
+
+After this, you can start working on the change.
```

Comment:
This is more of an expert mode workflow, really. I'm realizing that you might not have actually had enough context on basic mode to talk through the workflow. None of this is wrong, it's just too complicated for what we want to start with for someone with zero git background.

I'd honestly be fine with repurposing all you have as the expert mode documentation / as part of the coverage of Git WebUI. This is largely useful content still, just not for the bare-bones simplest-possible Health Connect Cloud use case.